### PR TITLE
WIP Onboard collab integration

### PR DESCRIPTION
**Pull request by piet0024**

### CODE REVIEW

File: src/containers/Onboard.js [hunk 1]
```diff
@@ -1,13 +1,174 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
 
-import { Container } from 'reactstrap';
+import { Mutation } from 'react-apollo';
+import { Container, Card, CardBody } from 'reactstrap';
+import { connect } from 'react-redux';
+
+import fetchJsonp from 'fetch-jsonp';
```
Comment (ethanWallace) [inline src/containers/Onboard.js:8]: My project is not able to find the fetch-jsonp package, was it added in this branch?

Comment (piet0024) [reply src/containers/Onboard.js:8]: Yes this was a new package I added. You will need to yarn install.

Comment (ethanWallace) [reply src/containers/Onboard.js:8]: You might need to commit the changes to the package.json into the PR still.

**Review by ethanWallace — CHANGES_REQUESTED** [review]
When a user enters the onboarding, the API will call and update the user's profile with information from GCcollab but the old information from the user's profile will still be show in the following steps of the onboarding.

File: src/containers/Onboard.js (hunk 1 above)
Comment (piet0024) [reply src/containers/Onboard.js:8]: Good catch! I will add it.

**Review by ethanWallace — APPROVED** [review]

**Pull request merged**